### ISSUE TITLE
"inspector" based debugging fails for node v6.5  

### ISSUE THREAD

**Issue by weinand** [Microsoft/vscode#25556]
- VSCode Version: 1.12.0 Insiders
- OS Version: macOS

Steps to Reproduce:

- use node v6.5
- create "test.js" containing `console.log("hello")`
- use this launch config:
  ```js
  {
	"type": "node",
	"request": "launch",
	"name": "Launch Program",
	"program": "${workspaceRoot}/test.js",
	"protocol": "inspector"
  }
  ```
- press "F5"

Observe:
![2017-04-27_16-40-59](https://cloud.githubusercontent.com/assets/1898161/25488845/5b6e894e-2b68-11e7-849b-01984c94bc04.png)

It works fine for node 7.4

**Comment by isidorn** [comment]
Does not look important based on @weinand

**Comment by weinand** [comment]
yes, because we are enabling the inspector protocol only for node.js versions >= 8.0 and the problem only occurs around version 6.5. So only users that enable the inspector protocol explicitly for 6.5 will run into it. Since they know what they doing, they can easily avoid it by upgrading to a newer version of node.js.

**Comment by roblourens** [comment]
Annoying, I thought `Schema.getDomains` was an old API. Easy fix but I agree, since it impacts 6.5 but not even 6.6. I recommend using at least 6.8 for stability.

**Comment by michelkaporin** [comment]
The error is not thrown any more, however debugging session terminates after start. Created a separate issue for that: #27929
